lokerpc/client_test.go

```
@@ -31,14 +37,97 @@ func TestNewClientNormalizesBaseURL(t *testing.T) {
 			}
 		})
 	}
 }
+
+func TestClient_DoRequest(t *testing.T) {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/notFound", http.NotFound)
+	mux.HandleFunc("/echo", func(w http.ResponseWriter, r *http.Request) {
+		body, _ := io.ReadAll(r.Body)
+		w.Header().Set("Content-Type", "application/json")
+		w.Write(body)
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)